### ISSUE TITLE
"BandagedBD is Loading..." never disappears

### ISSUE THREAD

**Issue by gotmilk0112** [Jiiks/BetterDiscordApp#901]
Not sure why, but the "BandagedBD is Loading..." flashing icon never disappears. I looked up a previous Issue for this and tried the solution (deleting user.js, deleting bdstorage.js) but it didn't work.

**Comment by RyderBelserion** [comment]
Did you try reinstalling?

**Comment by gotmilk0112** [comment]
Yeah, got the newest installer and reinstalled with that, didn't fix it.

**Comment by luni3359** [comment]
are you on linux? it happens to me as well

**Comment by gotmilk0112** [comment]
No, win10.

**Comment by luni3359** [comment]
Does this still happen to you? It should be fixed by now.
